wd_helper.au3

```diff
@@ -1489,12 +1512,21 @@ Func _WD_GetBrowserPath($sBrowser)
 		; Generate $_WD_ERROR_NotFound if neither key is found
 		If @error Then
 			$iErr = $_WD_ERROR_NotFound
+			$iExt = 22
 		Else
 			$sPath = StringRegExpReplace($sPath, '["'']', '') ; Remove quotation marks
 			$sPath = StringRegExpReplace($sPath, '(.+\\)(.*exe)', '$1' & $sEXE) ; Registry entries can contain "Launcher.exe" instead "opera.exe"
+
+			If FileExists($sPath) = 0 Then
+				$iErr = $_WD_ERROR_FileIssue
+				$iExt = 23
+			ElseIf _WinAPI_GetBinaryType($sPath) = 0 Then
+				$iErr = $_WD_ERROR_FileIssue
+				$iExt = 24
```

Comment:
I'm not sure how I feel about this. This intent of this function was to `Retrieve path to browser executable from registry` (taken directly from the header description). It doesn't state anything about validating that the resulting file path exists on disk.

Please explain why you think this code belongs here.

Reply:
Function name is `_WD_GetBrowserPath` 
When a function gives a path to a nonexistent file, it should mark it up somehow. The same if the file is not correct (for example not executable).
Everyone expects the function to return a valid working file (path to such a file).

btw. 
[You give me the reason](https://github.com/Danp2/au3WebDriver/pull/234#issuecomment-1055491099) to add such feature.
`The UDF shouldn't fail to return the corrrect value just because an arbitrary file with the same name exists in the current working directory.`

I hope you find my reasoning valuable.

Reply:
You may expect it, but I doubt that everyone does. 🙂 

Also, there is a difference between failing to return the correct value and performing checks beyond the scope of the function.

Reply:
If we are adding support for a portable version of browsers, we have to predict that the user will enter the wrong path. Wrong in any way. The UDF should report it so that you can easily find out the cause of the error. In this respect, it is not beyond the scope of this function.

Reply:
`_WD_GetBrowserPath` has nothing to do with supporting portable browsers, so unsure why you brought that up in the discussion. `_WD_GetBrowserVersion` is where these checks should be performed IMO, and the functionality is already in place.

Reply:
So take a look here: [WinAPI_GetBinaryType usage](https://github.com/Danp2/au3WebDriver/pull/235/commits/f2e41332e20c6674fae0520df863958624f0732f)